### PR TITLE
Replace explicit gendered terms with neutral

### PR DESCRIPTION
**Pull request by snickler**
Although there are only two instances referring to "him" and "himself" with regards to the user, it makes more sense to have something gender neutral in the verbiage.

### CODE REVIEW

File: 0004-oauth/README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The general framework is described in figure 1.
     ├───────────────────────────┬───────────────────────────────┼────────────────►│
     │                           │                               │                 ├──┐
     │                           │                               │                 │  │ Verification
-    │ User authenticates himself on AS and approves the request (10)              │◄─┘ (9)
+    │ User authenticates themshelf on AS and approves the request (10)              │◄─┘ (9)
```
Comment (teonbrooks) [inline 0004-oauth/README.md:100]:
```suggestion
    │ User authenticates themself on AS and approves the request (10)              │◄─┘ (9)
```

File: 0004-oauth/README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The general framework is described in figure 1.
     ├───────────────────────────┬───────────────────────────────┼────────────────►│
     │                           │                               │                 ├──┐
     │                           │                               │                 │  │ Verification
-    │ User authenticates himself on AS and approves the request (10)              │◄─┘ (9)
+    │ User authenticates themself on AS and approves the request (10)              │◄─┘ (9)
```
Comment (pfrazee) [inline 0004-oauth/README.md:100]:
```suggestion
    │ User authenticates themself on AS and approves the request (10)             │◄─┘ (9)
```

**Review by pfrazee — APPROVED** [review]
Thanks!

**Comment by 0x8008** [comment]
About as necessary as a code of conduct.

**Comment by snickler** [comment]
> About as necessary as a code of conduct. This is a code repository, not your local debate club and politics are about as necessary as Biden in the Presidential office.

Good thing this particular repository is for authoring proposals in markdown. If you read the changes in this PR, you would know that it affected a word in a sequence diagram and a paragraph to make the verbiage neutral. 

Carry on now, as this is merged and conversation is over.

**Pull request merged**